checkov/common/bridgecrew/integration_features/features/fixes_integration.py

```diff
@@ -112,8 +115,20 @@ def _get_fixes_for_file(
         if not self.bc_integration.http:
             raise AttributeError("HTTP manager was not correctly created")
 
-        logging.debug(f'Calling fixes API with payload: {json.dumps(payload)}, headers: {headers}, url: {self.fixes_url}')
-        request = self.bc_integration.http.request("POST", self.fixes_url, headers=headers, body=json.dumps(payload))  # type:ignore[no-untyped-call]
+        try:
+            logging.debug(f'Calling fixes API with payload: {json.dumps(payload)}, headers: {headers}, url: {self.fixes_url}')
+            request = self.bc_integration.http.request("POST", self.fixes_url, headers=headers, body=json.dumps(payload))  # type:ignore[no-untyped-call]
+        except ProtocolError as e:
+            logging.error(f'Get fixes request for file {filename} failed with response code error: {e}')
+            if isinstance(self.bc_integration.http, PoolManager):
```

Comment:
can you add a comment here explaining why this can help/when it happens

Reply:
added